Applications/Spire/Include/Spire/Ui/ColorSelectorHexInputWidget.hpp

```diff
@@ -28,15 +28,20 @@ namespace Spire {
       */
       void set_color(const QColor& color);
 
-      //! Connects a slot to the color signal.
-      boost::signals2::connection connect_color_signal(
+      //! Connects a slot to the color modified signal.
```

Comment:
I don't know of any signals that use the word modified. Can you look to rename this signal to bring it more in line with how Qt names it signals?

Reply:
Qt uses 'edited' for user changes, and 'changed' for both user changes and changes made programmatically. I used modified for the Key Bindings table signals, so I can update those as well.

Reply:
I think I'll go with Changed because it's more generic.